docs/resources/deployment-examples/example-fargate/terraform/task_def_ref.tf

```diff
@@ -0,0 +1,108 @@
+resource "aws_ecs_task_definition" "ref" {
+  network_mode             = "awsvpc"
+  requires_compatibilities = ["FARGATE"]
+  cpu                      = 256
+  memory                   = 512
+    family                   = "${var.environment}-ref"
+  execution_role_arn       = aws_iam_role.ecs_task_execution_role.arn
+  task_role_arn            = aws_iam_role.ecs_task_role.arn
+
+   volume {
+    name = "config"
+  }
+
+  container_definitions = jsonencode([{
+   name        = "${var.environment}-ref-config"
+   image       = "245943599471.dkr.ecr.us-east-2.amazonaws.com/anchor-platform-config:latest"
```

Comment:
Is this a public image?

Reply:
should be formed based on the codebuild like sep service. i fixed this.